src/jit/simdcodegenxarch.cpp

```diff
@@ -1144,12 +1144,11 @@ void CodeGen::genSIMDIntrinsicRelOp(GenTreeSIMD* simdNode)
                 // We need two additional XMM registers as scratch.
                 regMaskTP floatRsvdRegs = (simdNode->gtRsvdRegs & RBM_ALLFLOAT);
                 assert(floatRsvdRegs != RBM_NONE);
-                assert(genCountBits(floatRsvdRegs) == 2);
+                assert(genCountBits(floatRsvdRegs) == 1);
```

Comment:
I also need to fix the comment above this code, it still says that 2 registers are needed.